go.mod

```diff
@@ -1,6 +1,7 @@
 module go.dedis.ch/kyber/v3
 
 require (
+	github.com/btcsuite/btcd v0.0.0-20190410025418-9bfb2ca0346b
```

Comment:
btcd is a rather large dependency, since all we really want is btcec. Their license (the ISC license) would allow us to fork their code, and bring in btcec directly into Kyber if we wanted. btcec is fairly stable, having received just a couple commits in 2 years.

Did you consider incorporating btcec as source instead of as a dependency?

Reply:
No, I hadn't, but I'd be happy to do that if the licence allows, and expect it would be a straightforward modification.

Reply:
Yes, please.